workload-identity/README.md

```diff
@@ -43,7 +43,7 @@ bind-service-accounts.sh \
   SOMEBODY@PROJECT.iam.gserviceaccount.com
 ```
 
-This script assumes the same person can access both `K8S\_PROJECT` and
```

Comment:
This is to escape the _ character (as opposed to starting italics)

Reply:
Hmm I checked the preview page in https://github.com/kubernetes/test-infra/blob/deef8f2a358b813446f177a4d2f8c0d53fac9fbb/workload-identity/README.md#gcp-service-account and the formatting looks correct without the escape, so it's actually needed I think.

Reply:
Yeah, my editor must be doing something weird/wrong.